CHANGELOG

```diff
@@ -1,3 +1,12 @@
+smoke_test 1.0.3 (Jun 05, 2020)

```

Comment:
We'll do a new PyPI package release in a separate commit and git tag (to keep a single change per commit). If possible please *temporarily* remove this file change and the version bumps in `.py` files below. We'll add them back prior to new version release.

Other than that, this is ready to be merged.